app/components/UI/Bridge/components/BridgeDestNetworkSelector/index.tsx

```diff
@@ -13,6 +13,12 @@ import { VerticalAlignment } from '../../../../../component-library/components/L
 import { Hex, CaipChainId } from '@metamask/utils';
 import { BridgeNetworkSelectorBase } from '../BridgeNetworkSelectorBase';
 import { NetworkRow } from '../NetworkRow';
+import Routes from '../../../../../constants/navigation/Routes';
+import { selectChainId } from '../../../../../selectors/networkController';
+
+export interface BridgeDestNetworkSelectorRouteParams {
+  shouldGoToTokens: boolean;
```

Comment:
nit: this could be optional
```suggestion
  shouldGoToTokens?: boolean;
```

Reply:
Done in cbfefa1bbd7f719748ed9eca48dc0147773240ef